MergePythonSDK/ats/model/candidate.py

```diff
@@ -84,7 +84,7 @@ def additional_properties_type():
         This must be a method because a model may have properties that are
         of type self, this must run after the class is loaded
         """
-        return (bool, dict, float, int, list, str, none_type,)  # noqa: E501
+        return (bool, date, datetime, dict, float, int, list, str, none_type,)  # noqa: E501
```

Comment:
If memory serves these `date, datetime` types were removed intentionally but im not sure why